quantum/quantum.h

```diff
@@ -197,6 +197,8 @@ typedef uint8_t pin_t;
 
 #    define readPin(pin) ((bool)(PINx_ADDRESS(pin) & _BV((pin)&0xF)))
 
+#    define togglePin(pin) (writePin(pin, !readPin(pin)))
```

Comment:
I think the idea is good. But the implementation is not correct.

writePin() and readPin() access different I/O registers.